R/chrom-plots.R

```diff
@@ -118,6 +118,10 @@ binChroms = function(binCount, chromSizes) {
 #'     over; for example, these could be genome bins
 #' @return A data.table showing where on which chromosomes ranges are distributed.
 #' @export
+#' @examples
+#' vistaSftd = GenomicRanges::shift(vistaEnhancers, 100000)
```

Comment:
I don't think you can just add references like this to objects that don't exist in this context (`vistaEnhancers`), can you? does this run for you?